src/Components/Modal.js

```diff
@@ -0,0 +1,198 @@
+// import img from "./modal.jpg"
+// import { mdiWikipedia } from '@mdi/js';
+import {Icon} from '@iconify/react';
+
+export default function Modal({open, onClose, content}) {
+  console.log(content);
+  return (
+    open && (
+      <>
+        <Styled_Close_Button type="button" onClick={onClose}>
+          <Icon
+            icon="ant-design:close-square-outlined"
+            width="45"
+            height="45"
+          />
+        </Styled_Close_Button>
+        <StyledModalOuter>
+          <StyledModalInner>
+            <Styled_Wikipedia_Icon>
+              <Icon
+                icon="fa6-brands:wikipedia-w"
+                width="40"
+                height="40"
+                src={content.source}
+              />
+            </Styled_Wikipedia_Icon>
+            {/* images */}
+            <StyledImage src={content.spectral_img}></StyledImage>
+
+            <SteyledElement>
+              <span>{content.symbol}</span>
+            </SteyledElement>
+            <SteyledElementName>
+              <span>{content.name}</span>
+            </SteyledElementName>
+            <StyledAtomicMass>
+              <span>{content.atomic_mass}</span>
+            </StyledAtomicMass>
+
+            <Styled_Span>
+              <Styled_Icon>
+                <Icon icon="logos:react" width="25" height="25" />
+                <span>Overview</span>
+              </Styled_Icon>
+            </Styled_Span>
+
+            <span>NAME: {content.name}</span>
+            <span>ATOMIC MASS: {content.atomic_mass}</span>
+            <span>NAMED BY: {content.named_by}</span>
+            <span>CATEGORY: {content.category}</span>
+            <span>DISCOVERED BY: {content.discovered_by}</span>
+            <span>APPEARANCE: {content.appearance}</span>
+
+            <Styled_Span>
+              <Styled_Icon>
+                <Icon
+                  icon="fontisto:laboratory"
+                  color="red"
+                  width="25"
+                  height="25"
+                />
+                <span>Properties</span>
+              </Styled_Icon>
+            </Styled_Span>
+
+            <StyledLine>
+              <span>BOILING POINT: {content.boil}</span>
+            </StyledLine>
+
+            {/* //Line not working */}
+
+            <span>MELTING POINT: {content.melt}</span>
+            <span>DENSITY: {content.density}</span>
+            <span>SOURCE: {content.source}</span>
+            {/* <span className="name">{content.molar_heat}</span> */}
+            <span>ATOMIC NUMBER: {content.number}</span>
+            <span>PERIOD: {content.period}</span>
+            <span>PHASE AT STP: {content.phase}</span>
+            <span>SUMMARY: {content.summary}</span>
+            <span>{content.shells}</span>
+            <span>
+              ELECTRONIC CONFIGURATION: {content.electron_configuration}
+            </span>
+            {/* <span className="name">{content.electron_configuration_semantic}</span> */}
```

Comment:
Please delete these comments.

```suggestion

            <span>MELTING POINT: {content.melt}</span>
            <span>DENSITY: {content.density}</span>
            <span>SOURCE: {content.source}</span>
            <span>ATOMIC NUMBER: {content.number}</span>
            <span>PERIOD: {content.period}</span>
            <span>PHASE AT STP: {content.phase}</span>
            <span>SUMMARY: {content.summary}</span>
            <span>{content.shells}</span>
            <span>
              ELECTRONIC CONFIGURATION: {content.electron_configuration}
            </span>
```